docs/topics/modules/creating_using_new_work_item.adoc

```diff
@@ -21,7 +21,7 @@ image::new_wi.png[New Work Item]
 +
 image::wi_edit.png[Edit Work Item]
 +
-. Click *Add Assignee* and add your name as the new assignee for this Work Item. Click *X* to close the menu and save changes.
+. Click *Add Assignee* and add your name as the new assignee for this Work Item. Click *X* to close the menu and save changes. Assigning a Work Item sends that person an email notification. Depending on your mail server and account, this notification email may arrive after a delay of several minutes. 
```

Comment:
This is really sad... Are we expected to tell users they shouldn't worry because emails sometimes take some time to get delivered? I'm so looking forward to getting rid of this c*ap.

Reply:
I actually haven't ever received an email when testing this workflow so I hadn't mentioned it at all but Todd's doc asserts that it happens, so...

Reply:
@mishaone No notification is sent to the user doing the action. So unless you are a collaborator/commenter/issue creator in a space and someone else does a change, you won't see it. Normally, there is no real delay. A delay could happen as a new users domain goes through a whitelist process etc, but that's more of a first time thing from someone on that domain. There was a delay in the beginning of the notification email life, but that was due to low quota on the account and things queued up sending 25 emails an hour.

Reply:
Thanks, @aslakknutsen. That explains why I've never gotten an email.